gsd-schema/osv-gsd/README.md

```diff
@@ -0,0 +1,76 @@
+# OSV GSD Schema
+
+The GSD project primarily relies on the OSV schema for data, but we use an enhanced version of the currently released OSV schema. The reason for this is that the OSV takes time to release new versions of the schema, and in order to prevent blocking the release of data, we use the extended version which is backward compatible with the OSV schema. Using the standard OSV parser will result in warnings (e.g. some ENUM values we use are not present in OSV), but simply ignore the extended fields and attributes you will be able to process the data.
+
+# OSV GSD Extensions
+
+## severity["type": "CVSS_V2"]
+
+Upstream: https://github.com/ossf/osv-schema/pull/87
+Released in OSV schema: 1.4.0
+
+We needed CVSS v2 data in addition to CVSS v3 data so we can import historical data.
+
+## {"references": ["type": "DISCUSSION"]}
+
+Upstream: https://github.com/ossf/osv-schema/pull/138
+Released in OSV schema: 1.5.0
+
+We needed a DISCUSSION reference type for discussions of the issues that do not include fixes/etc.
+
+## {"$id": "VALUE"}
+
+Upstream: https://github.com/ossf/osv-schema/pull/130
+Released in OSV schema: 1.5.0
+
+We need to explicitly know what type of data this is, as do downstream consumers.
+
+## {"affected": ["ranges": ["events": ["last_affected": "VALUE"]]]}
+
+Upstream: https://github.com/ossf/osv-schema/pull/147
+Released in OSV schema: 1.5.0
+
+We need to explicitly set what the last affected version is for software like the Linux Kernel.
```

Comment:
How is this distinct from fixed?

Reply:
last_affected is the last affected version, there may not be a fixed version yet (if ever).